bin/oscapd-evaluate

```diff
@@ -503,9 +508,16 @@ def main():
     scan_parser.add_argument(
         "--output", type=str, required=True,
         help="A directory where results will be stored in. There will be a "
-        "directory for each target created there with up to 3 files. 'json' "
-        "with json summary of the scan, cve.xml with CVE scan raw results and "
-        "std.xml with configuration compliance scan raw results."
+        "directory for each target created there with up to 4 files. 'json' "
+        "with json summary of the scan, cve.xml with CVE scan raw results, "
+        "arf.xml with configuration compliance scan raw results, and "
+        "fix.[sh|yml|pp] with a compliance remediation script."
+    )
+    scan_parser.add_argument(
+        "--fix_type", type=str,
+        choices=["bash", "ansible", "puppet"], default="bash",
```

Comment:
@mpreisler I think it shouldn't. I think that will bring problems for users of Atomic from the UX point of view. Please let me explain this:

If no fix script will be generated by default, users of atomic scan will have to somehow magically know that if they use OpenSCAP as a underlying scanner and they want to make a hardened image, they have to pass another custom argument in scanner args. They will have to read oscapd-evaluate man page, but it's hard to guess that the information is there if they know only Atomic and aren't experts in the underlying technology. That would look like
`atomic scan --harden --scan_type configuration_compliance --scanner_args profile=xccdf_org.ssgproject.content_profile_nist-800-171-cui,fix_type=bash rhel7` I wonder how should be the behavior if they would omit that `,fix_type=bash`.


Moreover, it doesn't make sense to me to be able to specify a different fix type from atomic, because now people usually don't run Ansible or Anaconda inside their image, but Bash is almost everywhere, so bash remediations look like the only option for the image hardening use-case. Also, it could be possible replaced in future by some flexible API designed for customizing containers.

Another way of solving this problem is to hard-code `--fix-type bash` into `/etc/atomic.d/openscap`. Which one do you prefer? What do you suggest?